### PR TITLE
Adds *flex

### PR DESCRIPTION
**Pull request by ghost**
enraging @Iamgoofball since 2017

:cl: More Robust Than You
add: *flex
/:cl:

### CODE REVIEW

**Comment by Thunder12345** [comment]
But what if I'm a grille and want to flex my abs

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -169,4 +169,8 @@
 		var/turf/T = loc
 		T.Entered(src)
 
+/datum/emote/living/carbon/human/flex
+	key = "flex"
+	message = "flexs [p_their()] [pick("arms", ((gender == FEMALE) ? "flat stomach" : "abs"), "legs")]."
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/emote.dm]: remove the gender shit

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Thunder12345** [comment]
Its already been reviewed dumb bot

**Comment by Repukan** [comment]
When you have no personality, replace it with a *rolex

**Review by Cyberboss — DISMISSED** [review]

**Comment by lzimann** [comment]
use `*custom` for this, there's no reason to add a new datum

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -169,4 +169,13 @@
 		var/turf/T = loc
 		T.Entered(src)
 
+/datum/emote/living/carbon/human/flex
+	key = "flex"
+	message = "flexes their muscles."
+
+/datum/emote/living/carbon/human/flex/select_message_type(mob/user)
+	. = ..()
+	var/mob/living/carbon/human/H = user
```
Comment (lzimann) [inline code/modules/mob/living/carbon/human/emote.dm]: remove the first two lines of this proc

File: code/modules/mob/living/carbon/human/emote.dm [hunk 3]
```diff
@@ -169,4 +169,13 @@
...
 		T.Entered(src)
 
+/datum/emote/living/carbon/human/flex
+	key = "flex"
+	message = "flexes their muscles."
+
+/datum/emote/living/carbon/human/flex/select_message_type(mob/user)
+	. = ..()
+	var/mob/living/carbon/human/H = user
+	return "flexes [H.p_their()] [pick("arms", "abs", "stomach", "legs")]."
```
Comment (lzimann) [inline code/modules/mob/living/carbon/human/emote.dm]: this only needs to be `return "flexes their [pick(..)"`

**Comment by Militaires** [comment]
flex on the hater

**Comment by Militaires** [comment]
also how does this enrage goofball i dont think you know who or what goof is and how much influence he has....

**Comment by Iamgoofball** [comment]
yeah this is implemented correctly, if albeit useless due to *custom

File: code/modules/mob/living/carbon/human/emote.dm [hunk 4]
```diff
@@ -169,4 +169,11 @@
 		var/turf/T = loc
 		T.Entered(src)
 
+/datum/emote/living/carbon/human/flex
+	key = "flex"
+	message = "flexes their muscles."
+
+/datum/emote/living/carbon/human/flex/select_message_type(mob/user)
+	return "flexes their [pick("arms", "abs", "stomach", "legs")]."
```
Comment (xxalpha) [inline code/modules/mob/living/carbon/human/emote.dm]: benis

**Comment by jughu** [comment]
YES MRTY MY HERO

**Comment by lzimann** [comment]
This can be achieved easily with `*custom` or `me`, a new emote datum is not needed at all

**Pull request closed without merging**